### DISCUSSION TITLE
User Group Title

### DISCUSSION THREAD

**Discussion by asnanmtakim** [codeigniter4/shield#941] [Q&A] [answered]
How to get the group title or description from AuthGroups.php for the active user?

**Comment by kenjis** [comment, marked as answer]
You can get the groups by https://codeigniter4.github.io/shield/references/authorization/#getgroups 
and can get titles from Config:
https://github.com/codeigniter4/shield/blob/f1c2150e54f6aa5108a6e2cc25ddb00a77c5d801/src/Config/AuthGroups.php#L34-L38
